_tests/integration/android_test.go

```diff
@@ -18,32 +18,28 @@ import (
 func TestAndroid(t *testing.T) {
 	var testCases = []helper.TestCase{
 		{
-			"sample-apps-android-sdk22",
```

Comment:
I added structure field names to all the test cases because a new field (`RelativeSearchDir`) was introduced, and I wanted to avoid listing an additional empty parameter for all the test cases.